src/components/EventBlocks/index.tsx

```diff
@@ -39,15 +39,128 @@ export default function EventBlocks({
   includeContent,
   canBeTabFocused = false,
   deviceHasHover = true,
-  selectedMeeting,
-  onSelectMeeting,
+  daysRef,
+  timesRef,
 }: EventBlocksProps): React.ReactElement | null {
+  const [tempStart, setTempStart] = useState<number>(event.period.start);
+
+  const tempStartRef = useRef<number>(event.period.start);
+  const tempDaysRef = useRef<string[]>([...event.days]);
```

Comment:
When a recurring event occurs more than once a week, the blocks are dragged in conjunction with each other and we also cannot drag them to a different day. We should be able to drag the blocks separately.

Reply:
I guess we could seperate the events if any single block is moved, but that would cause a lot of accidental seperations

Reply:
Ah, I see what you mean. Then this should be fine. We don't really want to separate events for now.